src/luarocks/cmd/list.lua

```diff
@@ -71,8 +71,10 @@ end
 function list.command(flags, filter, version)
    local query = queries.new(filter and filter:lower() or "", version, true)
    local trees = cfg.rocks_trees
+   local title = "Rocks installed for Lua "..cfg.lua_version
    if flags["tree"] then
       trees = { flags["tree"] }
+      title = "Rocks installed for Lua "..cfg.lua_version.." in " .. flags["tree"]
```

Comment:
Could just append to `title` here instead of creating afresh?

Reply:
Sure, why not (building sentences out of fragments in code irks me a bit, but we don't have i18n anyway)